### PR TITLE
refactor unit tests

### PR DESCRIPTION
**Pull request by brycahta**
Issue #, if available:
* N/A

Description of changes:
* Update unit tests to use `assert` pattern for consistency and clarity

Testing:
* `make unit-test`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/cli/cli_test.go [hunk 1]
```diff
@@ -23,27 +23,21 @@ import (
 )
 
 func TestShowError_NoError(t *testing.T) {
-	if cli.ShowError(nil, "Test error shown") {
-		t.Error("No error but returns true")
-	}
+	actualShowError := cli.ShowError(nil, "Test error shown")
+	th.Equals(t, false, actualShowError)
```
Comment (pdk27) [inline pkg/cli/cli_test.go:27]: `th.Assert(t, actualShowError == true, "No error from cli but returns true")`? 
With a better failure msg :)

File: pkg/config/config_test.go [hunk 2]
```diff
@@ -34,27 +31,15 @@ var testConfigFilePath = os.Getenv("HOME") + "/.simple-ec2/" + testConfigFileNam
...
 	testString := "unit test config"
 	testData := []byte(testString)
-	path, err := config.SaveInConfigFolder(testConfigFileName, testData, 0644)
-	if err != nil {
-		t.Fatal(err)
-	}
 
-	// Check if the path is correct
-	if *path != testConfigFilePath {
-		os.Remove(*path)
```
Comment (pdk27) [inline pkg/config/config_test.go:44]: is this required? I am assuming testHelper won't be running `os.Remove`.

Also, would we loose some of the clear messaging with these changes? E.g. `The config file path is incorrect` is clear when the user sees the error as opposed to something like these 2 values don't match.

**Review by pdk27 — APPROVED** [review]
Super nice! 🥇 
Adds so much clarity. Thanks!

File: pkg/config/config_test.go (hunk 2 above)
Comment (brycahta) [reply pkg/config/config_test.go:44]: * Good call, this is most deff required. I'll update in next revision
* Truee, I'll update with `assert` instead and keep original messaging

Comment (brycahta) [reply pkg/config/config_test.go:44]: On the larger scale, I think foregoing the exact messaging is worth it so we can use these helpers and make the tests clear+readable. Unit test should be isolated enough so if it does fail, then whomever needs to fix it just needs to know what was expected and can deep dive from there.

**Review by pdk27 — APPROVED** [review]
LGTM

**Pull request merged**